libs/ledger-live-common/src/families/ethereum/modules/erc1155.ts

```diff
@@ -124,10 +126,22 @@ const erc1155Transfer: ModeModule = {
    */
   fillOptimisticOperation(a, t, op) {
     op.type = "FEES";
-    op.extra = {
-      ...op.extra,
-      approving: true, // workaround to track the status ENABLING
-    };
+    op.nftOperations = t.tokenIds?.map((tokenId, i) => {
+      const { currencyId } = decodeAccountId(a.id);
+      const nftId = encodeNftId(a.id, t.collection || "", tokenId, currencyId);
+
+      return {
+        ...op,
+        id: `${nftId}-#-NFT_OUT-i0_0`,
```

Comment:
optimistic operation are supposed to have the same ID as their future operations in order for Live not to display "duplicated" operations (one for optimistic, one for receiving one). notably due to fact we have a retain mecanics that keep optimistic operations for e.g. 30mn during the "unstability" phase of the server instances not all having operations synced.

is that the case? what is this `#` part?

Reply:
Yup I wanted to have the right ID from the beginning, but the second part of the ID that we craft is the hash, and I don't think we already have it at this stage ? I'll take another look, maybe I missed it ?